### PR TITLE
Bug 1702519: rhcos: Bump to 410.8.20190425.0

### PR DESCRIPTION
**Pull request by cgwalters**
This includes https://github.com/openshift/pivot/pull/51

### CODE REVIEW

File: data/data/rhcos.json [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
+            "hvm": "ami-0e2b5d1dcfbcc0c8f"
         },
         "us-west-2": {
-            "hvm": "ami-01e2758deb3135fb0"
+            "hvm": "ami-02b30dab0946cf89d"
         }
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/",
-    "buildid": "410.8.20190412.1",
+    "baseURI": "https://releases-art-jenkins.cloud.paas.upshift.redhat.com/storage/releases/ootpa/410.8.20190425.0/",
```
Comment (cgwalters) [inline data/data/rhcos.json]: Oops, one sec

**Comment by wking** [comment]
Updated the PR subject so generated change-logs will link to [rhbz#1702519][1] (which this PR is part of fixing).

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1702519

**Comment by miabbott** [comment]
This should pin to `410.8.20190425.1`;  `410.8.20190425.0` doesn't have the updated `pivot`

**Comment by miabbott** [comment]
/hold cancel

My bad...looking at the wrong pipeline.  The ART pipeline has `410.8.20190425.0` which has the right version of `pivot`.

The RHCOS devel pipeline has `410.8.20190425.0` too, but does not have the correct `pivot` build.

**Comment by cgwalters** [comment]
Hmm...I don't have quite enough knowledge of upgrade failures to say whether this is novel here but I don't think it is.

/retest

**Pull request merged**